crates/config/src/config.rs

```diff
@@ -133,6 +133,8 @@ pub struct TestNodeConfig {
     pub preserve_historical_states: bool,
     /// State to load
     pub load_state: Option<PathBuf>,
+    /// Port the spawned L1 anvil node will listen on
+    pub l1_anvil_port: Option<u16>,
```

Comment:
Can we add a print statement to `print` that indicates what port L1 is running at.

Reply:
Done